Woof/Woof/ViewModel/OwnerMainTabViewModel.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+/// The view model for a owner main tab view, responsible for preparing and processing data for the view.
+final class OwnerMainTabViewModel: ObservableObject {
+    /// Resets the current owner role  in the app by default value `none`.
+    func resertCurrentOwnerRole() {
```

Comment:
```suggestion
    func resetCurrentRole() {
```

?

Reply:
Thanks, for the suggestion. The function was renamed.